__tests__/app.test.js

```diff
@@ -80,4 +80,57 @@ describe("Test /api/articles/:article_id", () => {
   });
 });
 
+describe("TEST /api/articles", () => {
+  test("GET 200 Should return an array of articles", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((article) => {
+          expect(typeof article).toBe("object");
+          expect(article.hasOwnProperty("author")).toBe(true);
+          expect(article.hasOwnProperty("title")).toBe(true);
+          expect(article.hasOwnProperty("article_id")).toBe(true);
+          expect(article.hasOwnProperty("topic")).toBe(true);
+          expect(article.hasOwnProperty("created_at")).toBe(true);
+          expect(article.hasOwnProperty("votes")).toBe(true);
+          expect(article.hasOwnProperty("article_img_url")).toBe(true);
+          expect(article.hasOwnProperty("comment_count")).toBe(true);
+        });
+      });
+  });
+});
 
+describe("TEST /api/articles/:article_id/comments", () => {
+  test("GET 200: should return an array of comments coming with article id 1", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((comment) => {
+          expect(comment.article_id).toBe(1);
+          expect(comment.hasOwnProperty("comment_id")).toBe(true);
+          expect(comment.hasOwnProperty("body")).toBe(true);
+          expect(comment.hasOwnProperty("author")).toBe(true);
+          expect(comment.hasOwnProperty("votes")).toBe(true);
+          expect(comment.hasOwnProperty("created_at")).toBe(true);
+        });
+        expect(body[0].created_at > body[1].created_at).toBe(true);
```

Comment:
this last assertion is random. You haven't described this behaviour in your test string.
Are you checking for order? This isn't enough if that's the case. You're just checking the first two are in order.